play/src/front/Phaser/Game/MapEditor/MapEditorModeManager.ts

```diff
@@ -440,4 +448,45 @@ export class MapEditorModeManager {
     public getScene(): GameScene {
         return this.scene;
     }
+
+    public claimPersonalArea() {
+        const areaDataToClaim = get(mapEditorAskToClaimPersonalAreaStore);
+        const userUUID = localUserStore.getLocalUser()?.uuid;
+        if (areaDataToClaim === undefined) {
+            console.error("No area to claim");
+            return;
+        }
+        if (userUUID === undefined) {
+            console.error("Unable to claim the area, your UUID is undefined");
+            return;
+        }
+        const areaPersonalPropertyData = areaDataToClaim.properties.find(
+            (property) => property.type === "personalAreaPropertyData"
+        );
+        if (!areaPersonalPropertyData) {
+            console.error("No area property data");
+            return;
+        }
+        const scene = gameManager.getCurrentGameScene();
+
+        if (scene === undefined) {
+            console.error("GameScene is undefined");
+        }
```

Comment:
Scene cannot be undefined because `gameManager.getCurrentGameScene()` never returns undefined.